apps/meteor/app/slashcommands-mute/server/unmute.ts

```diff
@@ -11,43 +10,45 @@ import { api } from '../../../server/sdk/api';
  * Unmute is a named function that will replace /unmute commands
  */
 
-function Unmute(_command: 'unmute', params: string, item: IMessage): void | Promise<void> {
-	const username = params.trim().replace('@', '');
-	if (username === '') {
-		return;
-	}
-	const userId = Meteor.userId() as string;
-	const unmutedUser = Users.findOneByUsernameIgnoringCase(username);
-	if (unmutedUser == null) {
-		return api.broadcast('notify.ephemeralMessage', userId, item.rid, {
-			msg: TAPi18n.__('Username_doesnt_exist', {
-				postProcess: 'sprintf',
-				sprintf: [username],
-				lng: settings.get('Language') || 'en',
-			}),
-		});
-	}
+slashCommands.add(
+	'unmute',
+	function Unmute(_command, params, item): void | Promise<void> {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.